src/test/java/org/sonatype/gradle/plugins/scan/common/DependenciesFinderTest.java

```diff
@@ -410,6 +410,30 @@ public void testGetDependencies_ResolvedArtifacts_WithError() {
     assertThat(list.get(0).getId().getComponentIdentifier().toString()).isEqualTo(COMMONS_COLLECTIONS_DEPENDENCY);
   }
 
+  @Test
+  public void testGetDependencies_ResolvedArtifacts_WithDoubleError() {
```

Comment:
Suggestion. Maybe we can rename this to be clearer what is being tested? e.g. testGetDependencies_ResolvedArtifacts_Skip_Unresolvable_Dependencies()

Reply:
> Suggestion. Maybe we can rename this to be clearer what is being tested? e.g. `testGetDependencies_ResolvedArtifacts_Skip_Unresolvable_Dependencies()`

Done: 67073e7b27d33ade3f14e05627408a38e954715d